frontend/src/app/plot/plot.component.ts

```diff
@@ -50,7 +50,7 @@ export class PlotComponent implements OnInit {
   }]
 
   // Plot Configurations.
-  plot_layout = { title: 'Add a new dataset.', legend: 'false' }
+  plot_layout = { title: 'Add a new dataset.', legend: 'false', height: 1000 }
```

Comment:
Does height depend on display size?

Reply:
Right now it is just 1000px, independent of display size. Currently working on a way to get this to be 100% rather than a set # of pixels. (Surprisingly I can't just pass in 100%, it requires a number type so I have to get around that requirement)